lib/components/card.js

```diff
@@ -80,10 +84,10 @@ export default {
         // We are building up the markup.
         if (slots().img) {
             childNodes.push(slots().img);
-        } else if (props.img) {
+        } else if (props.img && !props.imgBottom) {
             childNodes.push(
                 h(CardImg, {
-                    props: { top: props.imgTop, bottom: props.imgBottom },
+                    props: { top: props.imgTop, bottom: props.imgBottom, imgFluid: props.imgFluid },
```

Comment:
What about building the card-img first in a var, then push it in the if sections? Save on repeating some code.